clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -454,7 +455,7 @@ public enum Command {
   ),
   MONITOR_DATA_RECOVERY(
       "monitor-data-recovery", "Monitor data recovery progress for a group of stores",
-      new Arg[] { URL, STORES, DEST_FABRIC }, new Arg[] { INTERVAL }
+      new Arg[] { URL, STORES, DEST_FABRIC }, new Arg[] { INTERVAL, DATETIME }
```

Comment:
Thanks for adding this new argument. I'm thinking whether we should make the argument required.

For example, operator runs monitor-data-recovery for store1 but does not specify a datetime, then current timestamp is used. Store1's data recovery completed before this monitor command runs. Even though store1's data recovery is done, as store1's current version timestamp is older than the start timestamp of monitor command, the monitor will unnecessarily keep waiting for a new version for store1 . This will make operator mistakenly think that store1's data recovery hasn't started yet.

If there is any benefit of making the argument optional that I'm not aware of, please don't hesitate to let me know.

Reply:
Shuhui, thank you for your comments! I agree that if we allow `datetime` to be optional, then we could have some misuse cases as you mentioned. Let me change it to required argument.